### ISSUE TITLE
PL/pgSQL function public.versioning() line 87 at RAISE

### ISSUE THREAD

**Issue by Fentonfi** [nearform/temporal_tables#26]

**Comment by Fentonfi** [comment]
Hi We have used the versioning.sql in our environment on pg11 and its working perfectly. There is no issue in our Logical replication from Master to logical server.
When we upgrade the DBMaster and Logical DB to PG12, we see the below issue. 
2020-09-16 22:58:26.808 [13255] ERROR:  relation "xxxxx_timetravel" does not exist
2020-09-16 22:58:26.808 [13255] CONTEXT:  PL/pgSQL function public.versioning() line 87 at RAISE
2020-09-16 22:58:26.810 [24781] LOG:  background worker "logical replication worker" (PID 13255) exited with exit code 1

The steps that we performed is 
1/ Set logical replication between master and logicaldb.
2/ Checked if all data is replicated and executed the versioning SQL on and ensured that the versioning of tables is working fine.
3/ DROP function if exists versioning() CASCADE;
4/ upgrade the server from pg11 to pg12 on dbmaster and logicaldb
5/ Dropped subscription and recreated subscription for logical replication 
6/ Reconfigure versioning function

After performing this activity we see the  errors as mentioned above. Does any one have any clue for if the process is correct or not?

Thanks,
Fenton
